tests/lib/cmsis_dsp/matrix/src/main.c

```diff
@@ -3,61 +3,3 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-
```

Comment:
shall we delete `main.c` since it is now an empty file?

Reply:
Hi @stephanosio , it seems like we cannot remove the empty main.c otherwise the cmake cannot find a source(even this test is skipped). I think put a empty main.c is better than using a  "#ifdef CONFIG_CMSIS_DSP_FLOAT16" in xx16.c .
Any other suggestions? Thanks!

Reply:
Fair enough, there is not much we can do at this time since a CMake run is required for the twister to be able to filter.